src/pages/TransactionDetailsPage.vue

```diff
@@ -55,6 +59,25 @@ watch(
   },
   { immediate: true }
 );
+
+const transactionTab = ref<ftm.TabInstructions>('All');
+
+const shouldShowAll = computed(() => transactionTab.value === 'All');
+
+const listState = computed(() => ({
+  kind: transactionTab.value,
+  transaction_hash: transactionHash.value,
+}));
```

Comment:
Use `reactive`, again.
```suggestion
const listState = reactive({
  kind: transactionTab,
  transaction_hash: transactionHash,
});
```